tokio/src/io/stderr.rs

```diff
@@ -132,3 +133,14 @@ impl AsyncWrite for Stderr {
         Pin::new(&mut self.std).poll_shutdown(cx)
     }
 }
+
+impl Stderr {
+    /// Returns true if the descriptor/handle refers to a terminal/tty.
+    pub fn is_terminal(&self) -> bool {
+        self.std
+            .inner()
+            .inner()
+            .map(|stderr| stderr.is_terminal())
+            .unwrap_or_default()
```

Comment:
This will incorrectly return `false` if stderr is a terminal but there is an ongoing IO operation. Can we instead do this?
```suggestion
    /// Returns true if the descriptor/handle refers to a terminal/tty.
    pub fn is_terminal(&self) -> bool {
        std::io::stderr().is_terminal()
```